.github/workflows/env-setup/action.yaml

```diff
@@ -4,9 +4,7 @@ description: "Sets node version, init pnpm, restore cache"
 runs:
   using: "composite"
   steps:
-    - uses: pnpm/action-setup@v2
-      with:
-        version: 8
```

Comment:
Otherwise:
```
Error: Multiple versions of pnpm specified:
    - version 8 in the GitHub Action config with the key "version"
    - version pnpm@8.[15](https://github.com/interledger/open-payments/actions/runs/9794096762/job/27043408325?pr=485#step:3:17).6 in the package.json with the key "packageManager"
  Remove one of these versions to avoid version mismatch errors like ERR_PNPM_BAD_PM_VERSION  
```